plugins/auth-backend/src/service/router.ts

```diff
@@ -25,16 +28,27 @@ export interface RouterOptions {
 export async function createRouter(
   options: RouterOptions,
 ): Promise<express.Router> {
-  const logger = options.logger.child({ plugin: 'auth' });
   const router = Router();
+  const logger = options.logger.child({ plugin: 'auth' });
```

Comment:
for the other plugins, the backend itself is responsible for creating the child logger, not the plugin

Reply:
@freben this seems to be the way catalog-backend also get the logger? Maybe I am missing something?

Reply:
Ya what I meant was, who shall have the responsibility of calling `.child`

I'm thinking maybe it should consistently be the backend itself?

Reply:
Agree, we create the namespace logger in backend as well. Don't have a strong opinion on that.